source/_docs/configuration/securing.markdown

```diff
@@ -1,29 +1,45 @@
+* **Run [configuration checks](https://www.home-assistant.io/common-tasks/os/#configuration-check).** After making configuration changes, you should ensure your configuration is valid so it will operate as expected.
 
-If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about) the founders of Home Assistant. 
+## Maintenance
 
-Other options are to use TLS/SSL via the add-ons [Duck DNS integrating Let's Encrypt](/integrations/duckdns/) or [Let’s Encrypt](/docs/ecosystem/certificates/lets_encrypt/). 
-To expose your instance to the internet, use a [VPN](https://pivpn.dev/), or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/). Make sure to expose the used port in your router. 
+Your system is your own responsibility. You should perform regular [Software Updates](https://www.home-assistant.io/common-tasks/os/#update) along with maintenance to ensure your system is secure. Many of these tasks can be automated. Your specific setup will dictate additional tasks.
+
+* **Check [Home Assistant Alerts](https://alerts.home-assistant.io/).** An informed user is able to make better decisions than an uninformed user.
+* **Read the [Release Notes](https://github.com/home-assistant/operating-system/releases/).**  Breaking changes can occur during updates and the release notes are Home Assistant's way of notifying you of these changes.
+* **Perform [Snapshots](https://www.home-assistant.io/common-tasks/os/#snapshots).**  Snapshots are a quick way to get your system back up and running in the event of failure.
+* **Update  [Encryption Certificates](/docs/configuration/securing/#remote-access).** TLS certs expire on a regular basis. Once expired your browser will not recognize them as valid. |
+* **Ensure all add-ons are updated.**  Running old versions of add-ons can lead to exploitable vulnerabilities which are not present in current versions.
+* **Prune Integrations/add-ons.** Remove unused Integrations and add-ons to reduce your risk of exposure.
+
+## Remote Access
 
-### Extras for manual installations
+If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about) the founders of Home Assistant.
 
-As well as the above we advise that you consider the following to improve security:
+To expose your instance to the internet, use a [VPN](https://pivpn.dev/), or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/). Make sure to expose the used port in your router. Other options are to use TLS/SSL via the add-ons [Duck DNS integrating Let's Encrypt](/integrations/duckdns/) or [Let’s Encrypt](/docs/ecosystem/certificates/lets_encrypt/).
```

Comment:
are we comfortable recommending PiVPN? I am not familiar with it myself

Reply:
I'm not familiar with it either.    This was part of the page for a very long time.  Simply reorganized.     As for my review, I haven't checked that app out.  Although  I might recommend some SSH Tunnel options:
**For android**
* [SSH Tunnel](https://play.google.com/store/apps/details?id=id.brooboox.sshtunnelmaker) for older versions of Android, but it crashes on Android 11.
* [KPN Tunnel](https://play.google.com/store/apps/details?id=kpn.soft.dev.kpntunnel) which works but is difficult to set up and uses non-standard terminology. 

**For Linux and Windows**
* [various options](https://linuxize.com/post/how-to-setup-ssh-tunneling/), but each option requires setup each time.

Then we get into non-SSH options and it doesn't feel right to start talking about SSH options for tunneling without talking about standard PPTP, L2TP/IPSec/ OpenVPN, SSTP and others...  In the end, I decided the pre-established advice was best.

What is your suggestion?

Reply:
I missed the original version when I was reviewing, my mistake. Let's leave it as is, the focus of this PR is on making the doc more comprehensive and better organized, not making third party recommendations.